openedx/core/djangoapps/course_groups/tests/test_views.py

```diff
@@ -125,7 +125,7 @@ def create_cohorted_discussions(self):
             self.course,
             is_cohorted=True,
             discussion_topics=discussion_topics,
-            cohorted_discussions=cohorted_discussions
+            divided_discussions=cohorted_discussions
```

Comment:
I don't think so because this is related to the variable names in the views method, which are still the old ones. This will change in your PR.